e2e-tests/pitr-physical/run

```diff
@@ -172,7 +174,6 @@ main() {
 
 	check_recovery $backup_name_minio-1 date "${last_chunk}" "-2nd" "$cluster"
 
-	run_backup $backup_name_minio 2 logical
```

Comment:
why no need for logical anymore?

Reply:
it was there as a workaround for PBM bug that doesn't start PiTR routine until a logical backup is created even if there's a physical backup. now the bug is fixed and running a logical backup is not necessary